pkg/util/expectations/resource_version_expectation.go

```diff
@@ -36,6 +36,13 @@ func NewResourceVersionExpectation() ResourceVersionExpectation {
 	return &realResourceVersionExpectation{objectVersions: make(map[types.UID]*objectCacheVersions, 100)}
 }
 
+func NewReallyNewerResourceVersionExpectation() ResourceVersionExpectation {
```

Comment:
plz add ut for the new expectation implementation

Reply:
implementation removed